src/main/scala/org/globalforestwatch/treecoverloss/TreeLossTile.scala

```diff
@@ -18,8 +18,8 @@ case class TreeLossTile(
   gain: Tile,
   tcd2000: Tile,
   tcd2010: Tile,
-  co2Pixel: Tile,
-  gadm36: Tile
+  co2Pixel: Option[Tile],
+  gadm36: Option[Tile]
```

Comment:
Another option here is to add `optionalRaster: Map[String, Tile]`

and access it as:
```scala
val maybeCo2: Option[Tile] = optionalRasters.get("co2Pixel")
```

this is more flexible but more likely to run into trouble with mistyped key. So you may want to put them into constants like
```scala
object TreeLossTile {
  final val CO2 = "co2"
}

...
val maybeCo2: Option[Tile] = optionalRasters.get(TreeLossTile.CO2)
```